webapp/src/css/dropdown.less

```diff
@@ -96,6 +96,7 @@
   }
 
   &.multidropdown .dropdown-menu {
+    min-width: 260px;
```

Comment:
Why do we have the multidropdown set to 260 but the regular dropdown just 240? Can we change line 84 to 260px and remove this line?

Reply:
`multidropdown` needs 20px extra because the ☑️ check at the left of the text, so I did not want to add to much extra space to the dropdown component when is not used with checks, but yes we can use just one width for both if you think is better.

Reply:
Ahh of course. Yes I think we should just make all the dropdowns the same size and the non-multidropdowns can just use the extra 20px for content.

Reply:
Yes it does not look bad with the extra 20px. Fixed.